src/dsql/ExprNodes.cpp

```diff
@@ -194,6 +192,55 @@ static void setParameterInfo(dsql_par* parameter, const dsql_ctx* context);
 //--------------------
 
 
+int SortValueItem::compare(const dsc* desc1, const dsc* desc2)
+{
+	return MOV_compare(JRD_get_thread_data(), desc1, desc2);
+}
+
+LookupValueList::LookupValueList(MemoryPool& pool, const ValueListNode* values, ULONG impure)
```

Comment:
Wouldn't be possible to avoid `const_casts` replacing some constructors to not use pointer to consts?

Reply:
I will try.